src/costmap/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+cmake_minimum_required(VERSION 3.5)
+project(costmap)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake REQUIRED)
+# find_package(rclcpp REQUIRED)
+# find_package(geometry_msgs REQUIRED)
+# find_package(std_msgs REQUIRED)
+find_package(nav2_msgs REQUIRED)
+find_package(tf2_ros REQUIRED)
+find_package(tf2 REQUIRED)
+
+include_directories(include)
+
+add_library(costmap STATIC
```

Comment:
We may want a general utility library that can hold Costmap, etc., instead of proliferating directories for each library.

Reply:
Agree, made a note to discuss this on our next meeting.

Reply:
As discussed, i moved it to a util library